contrib/docker-integration/test_runner.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+set -e
+
+cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+# Load the helpers.
+#. helpers.bash
+
+TESTS=${@:-.}
+
+# Drivers to use for Docker engines the tests are going to create.
+STORAGE_DRIVER=${STORAGE_DRIVER:-overlay}
```

Comment:
Second thought, will leave as overlay, added a section in the readme which mentions how to use aufs.